dev/AccessControl/Microsoft.Windows.Security.AccessControl.cpp

```diff
@@ -0,0 +1,50 @@
+﻿#include <pch.h>
+#include "Microsoft.Windows.Security.AccessControl.h"
+#include "Microsoft.Windows.Security.AccessControl.SecurityDescriptorHelpers.g.cpp"
+#include "Security.AccessControl.h"
+#include <sddl.h>
```

Comment:
NIT: Move to line 1.5

Unless it's already included in pch.h (somewhat common functionality)?